packages/ingest/HttpProviderClient.js

```diff
@@ -28,16 +28,14 @@ const validateHost = (host) => {
 
   throw new TypeError(`provider.host is not a valid hostname or IP: ${host}`);
 };
-const HTTP_REQUEST_TIMEOUT_SECONDS = 5 * 60; // Timeout for simplecrawler's headers response
 
 class HttpProviderClient {
   constructor(providerConfig) {
     this.providerConfig = providerConfig;
     this.protocol = providerConfig.protocol;
     this.host = providerConfig.host;
     this.port = providerConfig.port;
-    this.httpListTimeout = get(providerConfig,
-      'httpListTimeout', HTTP_REQUEST_TIMEOUT_SECONDS);
+    this.httpListTimeout = get(providerConfig, 'httpListTimeout');
```

Comment:
```suggestion
    this.httpListTimeout = providerConfig.httpListTimeout;
```

Reply:
Ack, right, undef will result in an exception regardless.  thanks.   👍